lib/matplotlib/axes/_axes.py

```diff
@@ -973,20 +975,19 @@ def hlines(self, y, xmin, xmax, colors='k', linestyles='solid',
             xmax = [xmax]
 
         y = np.ravel(y)
-
         xmin = np.resize(xmin, y.shape)
         xmax = np.resize(xmax, y.shape)
 
         verts = [((thisxmin, thisy), (thisxmax, thisy))
                  for thisxmin, thisxmax, thisy in zip(xmin, xmax, y)]
-        coll = mcoll.LineCollection(verts, colors=colors,
-                                    linestyles=linestyles, label=label)
-        self.add_collection(coll, autolim=False)
-        coll.update(kwargs)
+        lines = mcoll.LineCollection(verts, colors=colors,
+                                     linestyles=linestyles, label=label)
+        self.add_collection(lines, autolim=False)
+        lines.update(kwargs)
 
         if len(y) > 0:
-            minx = min(xmin.min(), xmax.min())
-            maxx = max(xmin.max(), xmax.max())
+            minx = np.min([xmin.min(), xmax.min()])
+            maxx = np.max([xmin.max(), xmax.max()])
```

Comment:
Originally, `vlines` used the builtin `min` and `max`, while `hlines` used a mix of builtin and numpy. I was going for consistency.

To my eyes, the numpy syntax is more familiar and it feels "safer" (not sure if that's true). I wasn't considering performance with those changes. 

My understanding is also that it's faster:

`x = numpy.ranndom.normal(size=3700)`
`%timeit x.max()`
> The slowest run took 53.27 times longer than the fastest. This could mean that an intermediate result is being cached.
> 100000 loops, best of 3: 2.88 µs per loop
where `(2.9 * 55 ~ 160 µs)`

`%timeit max(x)`
> 1000 loops, best of 3: 231 µs per loop